### ISSUE TITLE
Select Hours

### ISSUE THREAD

**Issue by hiralpatel** [Kunzisoft/Android-SwitchDateTimePicker#38]
While calendar view shows with a selected current date if I didn't choose a date and directly press the ok button the time picker view is not showing. How can I do that for show selection for a time?

**Comment by J-Jamet** [comment]
I don't really understand what you mean. When you pressed the " ok " button, the dialog dismiss, you get the DateTime and that's all we need. If you want to open the dialog again, open it in your code.

**Comment by hiralpatel** [comment]
Yes I get date and time always, but the scenario is like. When the date picker shows, without clicking/selecting date if we press "OK" button it is not showing Time picker. I want to show time picker compulsory after date picker.

**Comment by J-Jamet** [comment]
The purpose of the library is to show the date, time and year in a single dialog box, the ok button validates the entire dialog box and indicates that the user is happy with the year, the date and the time indicated in the header. If he wants to change a value, he passes between the views by clicking on the button of the header or on a label. In any case, the ok button will validate the information displayed, which is normal. If you want to manage the views separately, use the elements provided by android instead.

**Comment by praveenb** [comment]
just incase, If any one looking for Date and time selection mandatory, I've implemented changes [here](https://github.com/praveenb/Android-SwitchDateTimePicker)
